vdev/src/commands/integration/ci_paths.rs

```diff
@@ -0,0 +1,30 @@
+use anyhow::Result;
+use clap::Args;
+
+use crate::testing::config::IntegrationTestConfig;
+
+/// Output paths in the repository that are associated with an integration.
+/// If any changes are made to these paths, that integration should be tested.
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(&self) -> Result<()> {
+        // changes to vector-core should test all integrations
+        println!("all-int:");
+        println!("- \"lib/vector-core/**\"");
```

Comment:
I wonder if `vector-common` and `vector-config*` should also go in there.

Reply:
hmm let me find where this came up before. I feel like it was discussed.

Reply:
Here is where this was discussed before:

https://github.com/vectordotdev/vector/pull/17340#discussion_r1188949599

https://github.com/vectordotdev/vector/pull/17340#discussion_r1188945939

So my take on it is that yeah, I don't have a great grasp on whether those two should also go there. Would defer to your judgement, I'm happy to add them.

Reply:
Since this logic is unchanged from the previous state (AFAICT) I'd say just leave it for now.